core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -60,6 +62,14 @@ oppia.factory('SimpleEditorManagerService', [
       return 'Question ' + minimumStateNumber;
     };
 
+    var makeStateTerminal = function(stateName) {
```

Comment:
No, it's not exactly saving a state. It overwrites values so that they match properties of a terminal state.